.github/CODE_OF_CONDUCT.md

```diff
@@ -15,45 +15,74 @@ NB there is further documentation colocated with relevant code. The above list i
 
 ## Our Pledge
 
-In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+We as members, contributors, and leaders pledge to make participation in our community a harassment-free experience for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming, diverse, inclusive, and healthy community.
 
 ## Our Standards
 
-Examples of behavior that contributes to creating a positive environment include:
+Examples of behavior that contributes to a positive environment for our community include:
 
-- Using welcoming and inclusive language
-- Being respectful of differing viewpoints and experiences
-- Gracefully accepting constructive criticism
-- Focusing on what is best for the community
-- Showing empathy towards other community members
+- Demonstrating empathy and kindness toward other people
```

Comment:
While I'm not entirely opposed to mentioning empathy, opening with it as an example is very excluding to people like me who cannot practice it. So while I know what you mean can you say what you mean without this word?

In this case cutting "empathy and" changes little of the meaning.

re the remainder of the list, it's adequate but could meet plain English type guidelines better.

Reply:
- Respect different opinions, perspectives, and experiences
- Giving and appreciating constructive feedback
- Accepting responsibility and apologizing to those affected by our mistakes, and learning from the experience
- Focusing on what is best for us as individuals and the overall community
- Demonstrating kindness toward other people